citystreets/urls.py

```diff
@@ -25,6 +25,7 @@
     path("search/", include("search.urls")),
     path("map/", include("map.urls")),
     path("location/", include("location.urls")),
+    path("autofetch/", include("autofetch.urls")),
```

Comment:
Is there a more intuitive name than `autofetch`?
I read the code and what autofetch does. Personally, it doesn't seem to be `auto`? 🤔

Reply:
Well... Yes!
After reading your comment, maybe name it *background fetch* would make more sense, though it's longer

Reply:
It sounds better. However, from the django's perspective, I don't think it's a background fetch. django just receives `autofetch` request as same as other requests and it renders some page.

To me, it sounds like a typical view except the fact that it involves a huge update on our database.

Reply:
In my opinion, `auto` is done outside of the django app. It can be cronjob or celery(?) or something that I don't know. But it's clear that triggering the periodic fetching will not be included in our django application source code.

That's one of the reason I think why `auto` seems to be not very well matched with this view.

Reply:
I kinda agree with Yonguk here, the name doesn't really match what the function is actually doing and could potentially confusing to someone working on this down the line.  They may look at the name of the function and expect that it is something that runs automatically at a set time interval, whereas that's not really the case.

Perhaps something like `/refresh` or something similar?

Reply:
@alldne @ab7289 Renamed.